### PR TITLE
Update label based checks for pull requests

### PR DESCRIPTION
**Pull request by ankita240796**
## Explanation
- Updates changelog check such that a default label is added for non-collaborators.
- Updates assignee check such that comment is not created if an user is already assigned to a pull request.

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

I haven't been able to test this locally since my credentials are not working as expected. I am checking that to find the issue. @jameesjohn would it be possible to test these two features locally in the meantime?

UPDATE: Tested [here](https://github.com/ankita240796/oppiabot-test/pull/88).

### CODE REVIEW

File: spec/apiForSheetsSpec.js [hunk 1]
```diff
@@ -59,6 +59,10 @@ describe('Api For Sheets Module', () => {
           data: commitsData
         }),
       },
+      repos: {
+        checkCollaborator: jasmine.createSpy(
+          'checkCollaborator').and.returnValue(true),
+      }
```
Comment (jameesjohn) [inline spec/apiForSheetsSpec.js:65]: I'm not sure why this has been added here.

File: spec/checkPullRequestLabelsSpec.js [hunk 2]
```diff
@@ -199,7 +230,7 @@ describe('Pull Request Label Check', () => {
         body:
           'Hi, @' +
           payloadData.payload.pull_request.user.login +
-          '. This pull request does not have a "CHANGELOG: ..." label ' +
+          ', This pull request does not have a "CHANGELOG: ..." label ' +
```
Comment (jameesjohn) [inline spec/checkPullRequestLabelsSpec.js:233]: After the comma, the `This` should not start with capital letters.

File: spec/checkPullRequestLabelsSpec.js [hunk 3]
```diff
@@ -117,6 +127,27 @@ describe('Pull Request Label Check', () => {
         expect(github.issues.addAssignees).not.toHaveBeenCalled();
       });
 
+    it('should not assign project owner if they are already assigned',
```
Comment (jameesjohn) [inline spec/checkPullRequestLabelsSpec.js:130]: I don't think it's possible to assign a person when they are already assigned to an issue.

File: spec/apiForSheetsSpec.js (hunk 1 above)
Comment (ankita240796) [reply spec/apiForSheetsSpec.js]: Updated - created spy calls instead of this.

File: spec/checkPullRequestLabelsSpec.js (hunk 2 above)
Comment (ankita240796) [reply spec/checkPullRequestLabelsSpec.js]: Updated.

File: spec/checkPullRequestLabelsSpec.js (hunk 3 above)
Comment (ankita240796) [reply spec/checkPullRequestLabelsSpec.js]: Yes, my bad. The purpose is to ensure that comment is not added. Updated.

**Pull request merged**